graphene_sqlalchemy/converter.py

```diff
@@ -256,3 +259,40 @@ def convert_json_to_string(type, column, registry=None):
 @convert_sqlalchemy_type.register(JSONType)
 def convert_json_type_to_string(type, column, registry=None):
     return JSONString
+
+
+def _check_type(type_, args):
+    return type_ in (args + [x.__name__ for x in args])
+
+
+def convert_hybrid_type(type_, column):
```

Comment:
See my comment in the PR

Reply:
I need this feature, so I picked it up to try to finish the work.

Just as a heads up, I tried to use the actual `@singledispatch` mechanism and it is not compatible, since it actually extracts the type of the first argument, so for example, from a function's return type of `str`, this value correctly becomes `type` (being the type of the str class) :cry: 

I'm trying to code an approach similar to that where we can decorate functions and switch based on value.

Reply:
Let me know what you come up with! 😃

Reply:
I will, thank you :pray: I'll open a new Pull Request for #333 from my own branch as soon as I have the basics working.

Reply:
I want to finish this work. I will work on this PR tomorrow, so can it wait?

Reply:
I am branching by case statement because it returning the corresponding Graphene type in a class of `type` type obtained by `__annotations__`.
`functools.singledispatch` is a way to achieve polymorphism by instance type, but I don't know at this time how to achieve polymorphism by the class itself.

Reply:
I finished the work. I am opening a pull request against this issue.